### DISCUSSION TITLE
The "SMS/Text message" service in the "Two-factor methods" section is not working for Uzbekistan (+998)

### DISCUSSION THREAD

**Discussion by XushnazarovFaxriddin** [community/community#146226] [Other Feature Feedback, Questions, & Ideas]
The "SMS/Text message" service in the "Two-factor methods" section is not working for Uzbekistan (+998), I can't access the github mobile app, I'm not receiving an SMS code to my number, my number was +998936831555 but it changed to +9988936831555 by itself, this is an error, please change my number to +998936831555, I can't change it myself because Uzbekistan (+998) is missing among the country codes, my github username is @XushnazarovFaxriddin

**Comment by ghostinhershell** [comment]
Hi @XushnazarovFaxriddin 👋🏾 

Thank so much for writing in. If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

You can also find this, and other frequently asked questions, in our [Two Factor Authentication Frequently Asked Questions 🔑 🗝️](https://github.com/orgs/community/discussions/68240) post.

Let me know if you have any additional questions. Thanks!

**Comment by Mdshoyonmia** [comment]
Shoyon
